How-To-Setup-Internal-Package-Repository-Nexus.md

```diff
@@ -0,0 +1,241 @@
+
+    ![NuGet Repository Personal API Key in Nexus](images/nexus/nuget-repository-personal-api-key.png)
+
+Note that you can't push packages to a Proxy Repository as it doesn't support this.
+
+### Install A Package
+
+Let's install a package. As we have no packages in the repository as yet, any packages we do install will come from the URL we added as the **Remote Storage Location** [earlier](#create-a-proxy-repository-for-chocolatey-community-repository). Run `choco install winscp.portable --source <PROXY REPOSITORY PACKAGE SOURCE URL>`:
+
+```powershell
+> choco install winscp.portable --source "http://localhost:8081/nexus/service/local/nuget/proxy-repo/"
+
+Chocolatey v0.10.13
+
+winscp.portable v5.15.1
+winscp.portable package files install completed. Performing other installation steps.
+ ShimGen has successfully created a shim for WinSCP.exe
+ The install of winscp.portable was successful.
+  Software install location not explicitly set, could be in package or
+  default install location if installer.
+
+Chocolatey installed 1/1 packages.
+ See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
+```
+
+### What Packages Is Your Proxy Repository Storing
+
+So we installed the WinSCP package through the Proxy Repository so lets see the packages stored in the repository:
+
+1. Click the **Log In** button in the top right hand corner;
+1. If you have not changed the defaults then see **Step 1: Change the Administrative Password and Email Address** https://help.sonatype.com/repomanager3/installation/post-install-checklist for the defaults. **If the repository will be used in product we strongly recommend you change your username and / or password.**;
+1. Go to **Views/Repositories**;
+1. Click **Repositories** then click the **Proxy Repository** you created  [earlier](create-a-proxy-repository-for-chocolatey-community-repository);
+1. Under the **Browse Storage** tab you will see the `winscp.portable` package listed (note if you do not see the package click the **Refresh** button);
+  * Click the **+** button underneath to see the versions we have stored.
+
+  ![NuGet Proxy Repository Contents in Nexus](images/nexus/proxy-repository-contents.png)
+
+Any installation of the WinSCP package version we have stored, using the Proxy Repository as a source, will come directly from that repository and not from the Chocolatey Community Repository.
+
+## Saving Your Repository Source In Chocolatey
+
+In here we have explicitly passed the `--source` parameter to the Chocolatey commands. Remember you can instead [add the repository as a package source](https://chocolatey.org/docs/commands-sources) to avoid having to do that. Note however that you must still use the source explicitly when pushing a package.
```

Comment:
Could be nice to add a section pointing to the video @gep13 made: https://youtu.be/UehkG1VHtz0

Reply:
Is it worthwhile moving that video to the Chocolatey channel first?

/cc @gep13 @ferventcoder